### PR TITLE
#3645 - Content: Support Portal: Welcome Page

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Content changes;
- Removed "parents" panel when fulltime is not allowed.

image

Fulltime allowed:
image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/supportingusersdashboard.json [hunk 1]
```diff
@@ -1083,6 +1384,23 @@
...
       "id": "eiurv2i"
     },
+    {
+      "input": true,
+      "tableView": true,
+      "key": "isFulltimeAllowed",
+      "label": "isFulltimeAllowed",
+      "protected": false,
+      "unique": false,
+      "persistent": true,
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/supportingusersdashboard.json:1394]: Unless there is a need, hidden should be set as `"persistent": false`.

File: sources/packages/forms/src/form-definitions/supportingusersdashboard.json [hunk 2]
```diff
@@ -699,7 +998,7 @@
                           "disabled": false,
                           "tableView": false,
                           "modalEdit": false,
-                          "key": "startPartnerInformationRequestButton",
+                          "key": "startPartnerInformationRequestButtonFulltimeNotAllowed",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/supportingusersdashboard.json:1001]: Why was this change required?

**Review by andrewsignori-aot — COMMENTED** [review]
Nice work, minor comments only. Please take a look.

**Review by guru-aot — COMMENTED** [review]
LGTM, other than the key rename i am good to approve.

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by guru-aot — APPROVED** [review]
